pkg/oauth/apis/oauth/validation/validation.go

```diff
@@ -81,6 +81,8 @@ func ValidateAccessTokenUpdate(newToken, oldToken *oauthapi.OAuthAccessToken) fi
 	allErrs := validation.ValidateObjectMetaUpdate(&newToken.ObjectMeta, &oldToken.ObjectMeta, field.NewPath("metadata"))
 	copied := *oldToken
 	copied.ObjectMeta = newToken.ObjectMeta
+	// allow only TimeoutsIn to be changed
```

Comment:
TODO: Think about validation cases.

Reply:
checked that is not negative, that's all we can really do I think